src/ui/colorinputview.js

```diff
@@ -174,6 +175,7 @@ export default class ColorInputView extends View {
 		} );
 
 		dropdown.buttonView.children.add( colorPreview );
+		dropdown.buttonView.tooltip = t( 'Color picker' );
```

Comment:
Could you please add test for these? Something changed and we havn't documented it with tests.

Reply:
Added.